tide-compression/README.md

```diff
@@ -0,0 +1,16 @@
+# tide-compression
+
+This crate provides compression-related middleware for Tide.
+
+## Examples
+
+Examples are in the `/examples` folder of this crate.
+
+__Simple Example__
+
+You can test the simple example by running `cargo run --example simple` while in this crate's directory, and then running either of the following commands:
+
+```console
+$ curl http://127.0.0.1:8000/ -v
+$ curl http://127.0.0.1:8000/echo -v -d "why hello there"
```

Comment:
This command successfully tested compression and decompression.

```suggestion
$ echo 'why hello there' | gzip | curl -v --compressed -H 'Content-Encoding: gzip' 'http://127.0.0.1:8000/echo' --data-binary @-
```

Reply:
Oh, I totally forgot about the existence of the *actual* gzip binary... Woopsiedaisy. Thanks for your help!